app/serializers/sidebar_section_serializer.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class SidebarSectionSerializer < ApplicationSerializer
+  attributes :id, :title, :links, :slug
+
+  def links
+    object.sidebar_section_links.map(&:linkable)
```

Comment:
I believe this will result in an N+1 because I don't see `linkable` being preloaded.